### PR TITLE
Use new SDK and Python 3.7

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: .github/workflows/build-windows.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
         id: download-sdk
         uses: suisei-cn/actions-download-file@v1
         with:
-          url: https://github.com/freeorion/freeorion-sdk/releases/download/v12/FreeOrionSDK_12_MSVC-v141-Win32.zip
+          url: https://github.com/o01eg/freeorion-sdk/releases/download/v13/FreeOrionSDK_13_MSVC-v141-Win32.zip
```
Comment (geoffthemedio) [inline .github/workflows/build-windows.yml]: No reason not to use the v142 toolset SDK, I think...?

Comment (o01eg) [reply .github/workflows/build-windows.yml]: Travis CI builds SDK with v141 toolset.

**Review by Vezzra — APPROVED** [review]
As far as I can tell, the changes look ok. I can also successfully build and run this PR on macOS (meaning v13 of the SDK gets correctly pulled from the release in @o01eg's fork).

File: .github/workflows/build-windows.yml (hunk 1 above)
Comment (geoffthemedio) [reply .github/workflows/build-windows.yml]: It can be changed, I think? https://github.com/freeorion/freeorion-sdk/blob/master/.travis.yml#L75

Comment (o01eg) [reply .github/workflows/build-windows.yml]: Travis CI uses older Windows with `Visual Studio 15`.

**Comment by o01eg** [comment]
I've updated it to use SDK built in Github Actions.

**Review by Vezzra — COMMENTED** [review]
I can build FO successfully with this PR, but when I try to run it, it immediately crashes.
Output in terminal:
```
dyld: Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
  Referenced from: /Volumes/FreeOrion/FreeOrion.app/Contents/SharedSupport/libboost_iostreams.dylib
  Reason: image not found
Abort trap: 6
```

**Comment by o01eg** [comment]
@Vezzra Does it work with v13 which was built on Travis CI?

**Review by Vezzra — COMMENTED** [review]
> @Vezzra Does it work with v13 which was built on Travis CI?

If you mean the TravisCI SDK build you successfully managed to trigger and that I can download [here](https://github.com/o01eg/freeorion-sdk/releases/tag/v13), then yes, that SDK works. I can build and run FO with that.

**Comment by o01eg** [comment]
Error looks like Github Action CI links boost with system `zlib` but I think it better to switch to Travis CI generated SDK now.

**Comment by o01eg** [comment]
I've updated my `v14` and rebuild this PR with it.

**Review by Vezzra — DISMISSED** [review]
I can successfully build and run this PR.

**Comment by o01eg** [comment]
I made `v13` SDK release and switched this PR to it.

**Comment by Vezzra** [comment]
👍

**Pull request merged**